### DISCUSSION TITLE
Remove pure-visual settings from CL (Classic)

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#27622] [Ideas]
Options are "Always play a slider's tail sample" and "Fade out hit circles earlier" sounds like just osu ruleset customization and shouldn't be in CL mod IMO. If standardized vs classic scoring is in options, why those aren't? 
![image](https://github.com/ppy/osu/assets/89256026/a0b150c5-3d63-4aa4-8d4c-0dedb9f1d180)

**Comment by Rekunan** [comment]
The difference between the scoring options and this is that these directly affect gameplay, whereas scoring is just a gameplay stat that can be tweaked at any time.

**Reply by cl8n** [reply]
that is true but this poor example doesn't invalidate the issue imo. both of the mentioned CL options are almost completely preferential and I feel they give significantly less information about the score they were used on than say, the selected skin, or some other audio/visual settings. it's pointless clutter in the mod options and probably discourages people from enabling these settings independent of the other CL options
